frame/plasm-validator/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "pallet-plasm-validator"
+version = "0.1.0"
+authors = [
+    "Takumi Yamashita <takumi@stake.co.jp>",
+    "Aleksandr Krupenkin <aleksandr@stake.co.jp>"
+]
+edition = "2018"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "1.2.0", features = ['derive'], default-features = false }
+serde = { version = "1.0", optional = true }
+
+sp-core = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+sp-runtime = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+sp-arithmetic = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+sp-io = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+sp-std = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+pallet-balances = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+frame-support = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+frame-system = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+sp-staking = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master', default_features = false }
+
+pallet-plasm-rewards = { path = '../plasm-rewards', default_features = false }
+
+[dev-dependencies]
+hex-literal = "0.2.1"
+pallet-timestamp = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master' }
+pallet-session = { git = "https://github.com/staketechnologies/substrate.git", branch = "plasm-master" }
+pallet-staking = { git = 'https://github.com/staketechnologies/substrate.git', branch = 'plasm-master' }
+randomness-collective-flip = { package = "pallet-randomness-collective-flip", git = 'https://github.com/staketechnologies/substrate.git', branch = "plasm-master" }
```

Comment:
Unfortunately I can't find where random pallet is used.

Reply:
There are used by tests.

Reply:
pallet-staking is unused. thanks 🙏